### PR TITLE
Jest

### PR DESCRIPTION
**Pull request by rajusah15**
## Checklist

- [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: j/jest/jest_rhel8.3.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : jest
+# Version       : 9b9afadbbf7f0cf8f93d72656cba4b3288d0ae49 or tag: 25.2.3
+# Source repo   : https://github.com/DefinitelyTyped/DefinitelyTyped.git
+# Tested on     : UBI 8
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Raju.Sah@ibm.com
```
Comment (kandarpamalipeddi) [inline j/jest/jest_rhel8.3.sh:8]: Update the header as per Template.

File: j/jest/jest_rhel8.3.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : jest
+# Version       : 9b9afadbbf7f0cf8f93d72656cba4b3288d0ae49 or tag: 25.2.3
```
Comment (kandarpamalipeddi) [inline j/jest/jest_rhel8.3.sh:4]: `tag: 25.2.3` is not available.

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]
Please remove a/airline/airline_rhel_8.3.sh from this commit.

**Comment by rajusah15** [comment]
Will raise another PR as some unwanted file has uploaded and also follow the new template.

**Pull request closed without merging**